snapshot/lib/snapshot/simulator_launchers/simulator_launcher.rb

```diff
@@ -0,0 +1,140 @@
+require 'snapshot/simulator_launchers/simulator_launcher_base'
+
+module Snapshot
+  class SimulatorLauncher < SimulatorLauncherBase
+    def default_number_of_simultaneous_simulators
```

Comment:
Maybe we can add a comment here on why we do this? Would be good for future reference